server/SecondSurvey.cs

```diff
@@ -464,9 +464,11 @@ public static bool IsKnownT5Allegiance(string code)
             { "Gnii", "Gniivi" },
             { "Gray", "Graytch" },
             { "Guru", "Gurungan" },
+            { "Gurv", "Gurvin" },
```

Comment:
This table is auto-generated from the T5SS data drops (at least, the ones I used to get from Don). It'll get overwritten on the next one.